parsers/MD.py

```diff
@@ -72,17 +72,17 @@ class ArchiveDatapoint(NamedTuple):
     """Datapoint returned by the archive_url with ordered fetchable fields."""
 
     datetime: datetime
-    consumption: float
-    planned_consumption: float
-    production: float
-    planned_production: float
-    tpp: float  # production from thermal power plants
-    hpp: float  # production from hydro power plants
-    res: float  # production from renewable energy sources
-    exchange_UA_to_MD: float
-    planned_exchange_UA_to_MD: float
-    exchange_RO_to_MD: float
-    planned_exchange_RO_to_MD: float
+    consumption: float | None
+    planned_consumption: float | None
+    production: float | None
+    planned_production: float | None
+    tpp: float | None  # production from thermal power plants
+    hpp: float | None  # production from hydro power plants
+    res: float | None  # production from renewable energy sources
+    exchange_UA_to_MD: float | None
+    planned_exchange_UA_to_MD: float | None
+    exchange_RO_to_MD: float | None
+    planned_exchange_RO_to_MD: float | None
```

Comment:
note: it looks like the API can return null values, so I have updated the code to make sure we can handle them without crashing

e.g. https://moldelectrica.md/utils/archive2.php?id=table&type=array&date1=14.04.2024&date2=16.04.2024